coffee-module/coffee-module-redisstream/src/main/java/hu/icellmobilsoft/coffee/module/redisstream/consumer/AbstractStreamConsumer.java

```diff
@@ -57,12 +65,43 @@ public void onStream(StreamEntry streamEntry) throws BaseException {
             String flowId = fieldMap.getOrDefault(IRedisStreamConstant.Common.DATA_KEY_FLOW_ID, mainData);
             MDC.put(LogConstants.LOG_SESSION_ID, flowId);
 
+            if (customize(streamEntry)) {
```

Comment:
ezzel lehetne kezdeni a metodust, az mdc az nem kell ha csak ackolni fog nem?

Reply:
szandekosan ott van hagyva a kiskapu hogy projektek egyedi kezelest is tudjanak csinalni az uzenettel. E kozben logolhatnak, DB-t kezelhetik vagy csinalhatnak barmit, aminel nagyon jol jon a beallitott MDC